cmd/utils/flags.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE file in the project root for license information.
+
+package utils
+
+import "github.com/spf13/cobra"
+
+// CommonFlags contains CLI flags common for all subcommands
+type CommonFlags struct {
+	SubscriptionID string
+	ResourceID     string
+	Verbose        bool
+}
+
+func AddCommonFlags(command *cobra.Command, params *CommonFlags) {
+	command.PersistentFlags().StringVarP(
+		&params.SubscriptionID,
+		"subscription", "",
+		"",
+		"Subscription ID.",
+	)
+	command.PersistentFlags().StringVarP(
+		&params.ResourceID,
+		"id", "",
+		"",
+		`Complete resource ID containing all information of the resource,
+		e.g. /subscriptions/MySubID/resourceGroups/MyRes for a resource group resource.
+		Notice it is not case sensitive.`,
+	)
```

Comment:
That's true, it couldn't be needed for all the commands. However, I put there because it is generic as concept because it is not associated to a specific command as `vmss` (only for commands that refers to a node) or `node-resoure-group` (only for resource within the node resource group).

Reply:
I managed them in different way. Please let me know what do you think now.